libs/components/src/lib/button/button.template.ts

```diff
@@ -34,24 +34,19 @@ function renderIconOrPending(
 	pending: boolean,
 	size: ButtonSize | undefined = Size.Normal
 ) {
-	let content = '';
-	let classes = 'icon';
-
 	if (pending && size != Size.SuperCondensed) {
 		const progressTag = context.tagFor(ProgressRing);
 		const progressSize = {
 			[Size.Condensed]: '-6',
 			[Size.Normal]: '-5',
 			[Size.Expanded]: '-4',
 		};
-		content = `<${progressTag} size="${progressSize[size]}"></${progressTag}>`;
-		classes += ' pending';
-	} else if (icon) {
-		const iconTag = context.tagFor(Icon);
-		content = `<${iconTag} name="${icon}"></${iconTag}>`;
-	}
+		return html`<span class="icon pending">${`<${progressTag} size="${progressSize[size]}"></${progressTag}>`}</span>`;
```

Comment:
We can do it like this, right?
```suggestion
		return html`<span class="icon pending"><${progressTag} size="${progressSize[size]}"></${progressTag}></span>`;
```